### PR TITLE
Fixed Manifest link used in 2.13.0 docs

### LINKED ISSUE #3798: doc: wrong manifest link in docs
**What happened**:

Manifest link used in 2.13.0 docs is wrong should be :
```
https://raw.githubusercontent.com/litmuschaos/litmus/2.13.0/mkdocs/docs/2.13.0/litmus-2.13.0.yaml
```
but given
```
https://raw.githubusercontent.com/litmuschaos/litmus/2.13.0/docs/2.13.0/litmus-2.13.0.yaml
```

ref: https://docs.litmuschaos.io/docs/user-guides/uninstall-litmus/#using-kubectl

### PR DESCRIPTION
**Pull request by anirudh5harma**
Signed-off-by: Anirudh Sharma <anirudh1304@gmail.com>

**What this PR does / why we need it**:
Fixes Manifest link used in 2.13.0 docs

**Which issue this PR fixes** 
fixes https://github.com/litmuschaos/litmus/issues/3798

**Special notes for your reviewer**:

**Checklist:**
-   [ ] Fixes #<issue number>
-   [ ] Signed the commit for DCO to be passed
-   [ ] Labelled this PR & related issue with `documentation` tag

### CODE REVIEW

File: website/versioned_docs/version-2.13.0/user-guides/uninstall-litmus.md [hunk 1]
```diff
@@ -102,7 +102,7 @@ To uninstall the ChaosCenter from the system, follow these steps -
 - **Litmus 2.13.0**
 
   ```bash
-  kubectl delete -f https://raw.githubusercontent.com/litmuschaos/litmus/2.13.0/docs/2.13.0/litmus-2.13.0.yaml
+  kubectl delete -f https://raw.githubusercontent.com/litmuschaos/litmus/2.13.0/mkdocs/docs/2.13.0/litmus-2.13.0.yaml
   ```
 
   > To delete any specific version of the ChaosCenter, replace the above command with the below command. `kubectl delete -f https://raw.githubusercontent.com/litmuschaos/litmus/<VERSION>/mkdocs/docs/<VERSION/litmus-<VERSION>.yaml`
```
Comment (neelanjan00) [inline website/versioned_docs/version-2.13.0/user-guides/uninstall-litmus.md]:
```suggestion
  > To delete any specific version of the ChaosCenter, replace the above command with the below command. `kubectl delete -f https://raw.githubusercontent.com/litmuschaos/litmus/<VERSION>/mkdocs/docs/<VERSION>/litmus-<VERSION>.yaml`
```

**Review by neelanjan00 — CHANGES_REQUESTED** [review]

**Comment by neelanjan00** [comment]
Thanks for the PR @AnirudhBot !

**Review by gdsoumya — APPROVED** [review]

**Comment by gdsoumya** [comment]
Thanks for the fix @AnirudhBot 🚀

**Comment by neelanjan00** [comment]
@AnirudhBot, the typo in `<VERSION>` is there in the previous versions of the doc as well. Can you make the changes in all of them?

**Comment by anirudh5harma** [comment]
> @AnirudhBot, the typo in `<VERSION>` is there in the previous versions of the doc as well. Can you make the changes in all of them?

Yeah sure! If I am not wrong, this typo got introduced from 2.11.0 only

**Review by ksatchit — APPROVED** [review]
LGTM

**Review by neelanjan00 — APPROVED** [review]
LGTM

**Pull request merged**